R/catmaid_tags.R

```diff
@@ -0,0 +1,59 @@
+#' Functions for retrieving soma data
+#'
+#' @description Functions for retrieving data about somata from skeletons, that has been assigned in CATMAID using the tag system
```

Comment:
I would include in the `description` what each function does. I know you put this in the return section but it's harder to find. I normally use the @description multiple times next to each method where it is defined.

```
@description \code{soma} fetches the XYZ location of the soma
@description \code{somaidx} fetches the raw index of the skeleton node associated with the soma (in range 1..N, number of vertices)
@description \code{somaid} fetches the identifier of the skeleton node associated with the soma
```